src/main/java/org/folio/util/ResponseUtils.java

```diff
@@ -24,38 +23,40 @@ public class ResponseUtils {
   private ResponseUtils() {
   }
 
-  public static <T> Handler<AsyncResult<Tx<T>>> handleNoContentResponse(Handler<AsyncResult<Response>> asyncResultHandler, Tx<T> tx,
-    String logMessage) {
-    logger.debug("handleNoContentResponse:: Trying to handle no content response for entity: {}', logMessage: {}", tx.getEntity(), logMessage);
-    return result -> {
-      if (result.failed()) {
-        HttpException cause = (HttpException) result.cause();
-        logger.error(logMessage, cause, tx.getEntity(), "or associated data failed to be");
-
-        // The result of rollback operation is not so important, main failure cause is used to build the response
-        tx.rollbackTransaction().onComplete(res -> asyncResultHandler.handle(buildErrorResponse(cause)));
-      } else {
-        logger.info(logMessage, tx.getEntity(), "and associated data were successfully");
-        asyncResultHandler.handle(buildNoContentResponse());
-      }
-    };
+  public static <T> void handleNoContentResponse(AsyncResult<T> result,
+                                                 String id,
+                                                 Handler<AsyncResult<Response>> onComplete) {
+    if (result.failed()) {
+      HttpException cause = (HttpException) result.cause();
+      logger.error("Failed to delete interface '{}' or associated data", id, cause);
```

Comment:
maybe better to folow the same style as in other places, write logs in invoker methods, not in this common method, because in future when someone would like to reuse this method - but this log is very specific only to interfaces

Reply:
simplified farther. Also `failOnNoUpdateOrDelete` might be usefull because similar logic is used in many places